# OpenStax Chemistry, 2nd Ed. &ndash; Dean's Errata Submissions
## Chapter 5

**Execute this cell to display chemical formulas.**
It contains a LaTeX require for mhchem, which even applies to later cells.
$ \require{mhchem} $

### Chapter 5 - Formulas and Concepts

#### Chapter 5.1

$$ q_{substance\ a} + q_{substance\ b} = 0 $$
$$ \text{specific heat}\  c = \frac{q}{m\Delta T} $$
$$ \Delta T = T_{final} - T_{initial} = T_f - T_i $$
$$
\begin{align}
q &= c \cdot m \cdot \Delta T \\ 
&= c \cdot m \cdot (T_{final} - T_{initial})
\end{align}
$$
$$ (c_a \cdot m_a \cdot \Delta T_a) + (c_b \cdot m_b \cdot \Delta T_b) = 0 $$
$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) + (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) = 0 $$
$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) $$

#### Chapter 5.3

$$ \begin{align}
q &:= \text{heat (heat flow into system is positive)} \\
q_p &:= \text{heat under constant P} \\
w &:= \text{work (work done on system is positive, by system is negative)} \\
U &:= \text{internal energy} (\equiv E) \\
P &:= \text{pressure} \\
V &:= \text{volume} \\
H &:= \text{enthalpy} \\
\\
\Delta U &= q + w \\
H &= U + PV \text{ (unmeasurable)} \\
\Delta H &= \Delta U + \Delta (PV) \text{ (measurable)} \\
\Delta H &= \Delta U + P \Delta V \text{ (assuming constant P)} \\
P \Delta V &= -w \\
\\
\Delta H &= \Delta U + P \Delta V \text{ (assuming constant P)} \\
&= q_p + w + - w \\
&= q_p \\
\end{align} $$

### Chapter 5 &ndash; Examples

#### Example 5.3 Solution

Define $a$ to be the metal, and $b$ to be the water.
$$ T_{a\ final} - T_{a\ initial} = - \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) $$
$$ T_{a\ initial} = \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) + T_{a\ final} $$
$$ m_b = 425 g $$
$$ c_b = 4.184 \frac{J}{g {\degree}C} $$
$$ T_{b\ final} = 42.7 {\degree}C $$
$$ T_{b\ initial} = 24.0 {\degree}C $$
$$ m_a = 360.0 g $$
$$ c_a = 0.449 \frac{J}{g {\degree}C} $$
$$ T_{a\ final} = T_{b\ final} = 42.7 {\degree}C $$
$$ T_{a\ initial} = \left(\frac{425 g \cdot 4.184 \frac{J}{g {\degree}C} \cdot (42.7 {\degree}C - 24.0 {\degree}C)}{360.0 g \cdot 0.449 \frac{J}{g {\degree}C}}\right) + 42.7 {\degree}C $$

In [1]:
((425 * 4.184 * (42.7 - 24.0)) / (360.0 * 0.449)) + 42.7

248.41851026973524

$$ T_{a\ initial} = 248 {\degree}C $$

#### Example 5.3 Check Your Learning 1

$$ T_{a\ initial} = \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) + T_{a\ final} $$

In [2]:
((4.184 * 390 * (39.9 - 22.6)) / (0.385 * 248))

295.6582320904901

In [3]:
((4.184 * 390 * (39.9 - 22.6)) / (0.385 * 248)) + 39.9

335.5582320904901

336 degrees C

#### Example 5.3 Check Your Learning 2

$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) $$
Define $a$ to be the metal, and $b$ to be the water.
Note that $T_{a\ final}$ should always equal $T_{b\ final}$.
$$ (c_a \cdot m_a \cdot (T_{final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{final} - T_{b\ initial})) $$
$$ \require{cancel} $$
$$ \frac{\cancel{c_a \cdot m_a \cdot} (T_{final} - T_{a\ initial})}{\cancel{c_a \cdot m_a}(c_b \cdot m_b)} = - \frac{\cancel{c_b \cdot m_b \cdot} (T_{final} - T_{b\ initial})}{(c_a \cdot m_a)\cancel{c_b \cdot m_b}} $$
$$ \frac{T_{final} - T_{a\ initial}}{c_b \cdot m_b} = - \frac{T_{final} - T_{b\ initial}}{c_a \cdot m_a} $$
$$ \frac{T_{final}}{c_b \cdot m_b} - \frac{T_{a\ initial}}{c_b \cdot m_b} = - \frac{T_{final}}{c_a \cdot m_a} - \frac{T_{b\ initial}}{c_a \cdot m_a}$$

**Wait, back up**
$$
\begin{align}
c_a \cdot m_a \cdot (T_{final} - T_{a\ initial}) &=
- c_b \cdot m_b \cdot (T_{final} - T_{b\ initial})
\\[1 ex]
% c_a \cdot m_a \cdot T_{final} -
% c_a \cdot m_a \cdot T_{a\ initial} &=
% - c_b \cdot m_b \cdot T_{final} +
% c_b \cdot m_b \cdot T_{b\ initial}
% \\[1 ex]
c_a \cdot m_a \cdot T_{final} -
c_a \cdot m_a \cdot T_{a\ initial} &=
c_b \cdot m_b \cdot T_{b\ initial} -
c_b \cdot m_b \cdot T_{final}
\\[1 ex]
c_a \cdot m_a \cdot T_{final} +
c_b \cdot m_b \cdot T_{final} &=
c_b \cdot m_b \cdot T_{b\ initial} +
c_a \cdot m_a \cdot T_{a\ initial}
\\[1 ex]
(c_a \cdot m_a + c_b \cdot m_b) \cdot T_{final} &=
c_b \cdot m_b \cdot T_{b\ initial} +
c_a \cdot m_a \cdot T_{a\ initial}
\\[1 ex]
T_{final} &=
\frac{c_b \cdot m_b \cdot T_{b\ initial} + c_a \cdot m_a \cdot T_{a\ initial}}
     {c_a \cdot m_a + c_b \cdot m_b}
\end{align}
$$

Defining $a$ to be the copper and $b$ to be the water, we have...
$$
T_{final} =
\frac{4.184 J/g {\degree}C \cdot 390 g \cdot 22.6 {\degree}C +
      0.385 J/g {\degree}C \cdot 248 g \cdot 314 {\degree}C}
     {0.385 J/g {\degree}C \cdot 248 g + 4.184 J/g {\degree}C \cdot 390 g}
$$

In [4]:
((4.184 * 390 * 22.6) + (0.385 * 248 * 314)) / ((0.385 * 248) + (4.184 * 390))

38.70828373590237

38.7 degrees C

#### Example 5.4 Solution

$$ \begin{align}
\Delta T = T_{final} - T_{initial} &= T_f - T_i
\\[1ex]
c_{metal} \times m_{metal} \times \Delta T_{metal} &= -
c_{water} \times m_{water} \times \Delta T_{water}
\\[1 ex]
c_{metal} &= - \frac{c_{water} \times m_{water} \times \Delta T_{water}}
                    {m_{metal} \times \Delta T_{metal}}
\\[1 ex]
c_{metal} &= - \frac{4.184 J/g \degree C \times 60.0 g \times (28.5 - 22.0 \degree C)}
                    {59.7 g \times (28.5 - 100.0 \degree C)}
\end{align} $$

In [5]:
- (4.184 * 60.0 * (28.5 - 22.0)) / (59.7 * (28.5 - 100.0))

0.3822750114207401

Specific heat $c_{metal} = 0.382 J / g \degree C$, so the metal is probably Copper.

**<span style="color: red">Note:</span>** The book said 0.38 (and copper).  They had an interim step where (28.5 - 22.0) was reduced to (6.5).  I am guessing that explains why they rounded the answer to 2 significant digits.  I'm guessing that the rounding is arguable either way, because if you do the calculation all in one step you would round to 3 significant digits.

#### Example 5.4 Check Your Learning

In [6]:
- (4.184 * 75.0 * (29.7 - 24.0)) / (92.9 * (29.7 - 178.0))

0.12982876620355413

Specific heat $c_{metal} = 0.130 J / g \degree C$ for the silvery-gray metal.

The metal is probably lead, because the only other metal listed in Table 5.1 is gold, which does not have a silvery gray color.

<strong style="color: red">Note:</strong>  The book said 0.13 (and lead).  I am guessing resolving the (29.7 - 24.0) term resulted in rounding to 2 significant digits.

#### Example 5.5 Solution

The book gives -2.9 kJ as the answer but assumes:
- The density of the solutions are approximately the same as water, 1.0 g/mL
- The specific heat of the solutions are approximately the same as water 4.184 J / g &deg;C
- The calorimeter allows no heat to enter or leave.
I want to try removing the first two assumptions, and see if I can calculate a more specific answer.  I will assume that the solutions are not being added to any additional water.

So, the problem statement from the book is:

> When 50.0 mL of 1.00 M HCl(aq) and 50.0 mL of 1.00 M NaOH(aq), both at 22.0 &deg;C, are added to a coffee cup calorimeter, the temperature of the mixture reaches a maximum of 28.9 &deg;C. What is the approximate amount of heat produced by this reaction?

> $$ \ce{HCl(aq) + NaOH(aq) -> NaCl(aq) + H2O(l)} $$

$$ 50.0\ mL * 1.00\ mol/L = .0500 L * 1.00 mol/L = 0.500 mol HCl $$

<strong style="color:red">Wait.</strong> I do not know the density of aqueous solutions of HCl (a gas) or NaOH, nor do I know a good place to look them up, or calculate the densities at a particular molarity $M$.  I thought I could maybe calculate from the molar masses, but that does not help me with the densities.  Therefore, **nevermind**.


#### Example 5.5 Check Your Learning

When 100 mL of 0.200 M NaCl(aq) and 100 mL of 0.200 M AgNO<sub>3</sub>(aq), both at 21.9 °C, are mixed in a coffee cup calorimeter, the temperature increases to 23.5 °C as solid AgCl forms. How much heat is produced by this precipitation reaction? What assumptions did you make to determine your value? 

In [7]:
-((100 + 100) * 1.0 * 4.184 * (23.5 - 21.9))

-1338.8800000000012

$$ q_{reaction} = - q_{solution} = -1.3 kJ $$
We assumed that both input solutes were roughly the same density and specific heat as liquid water, and that the calorimeter did not absorb any heat or let any heat enter or escape.

**<span style="color:red">Note:</span>** The book said 1.34 kJ, which differs from the rounding behavior used in the Example 5.5 Solution just above the Check Your Learning, so I will report it.

#### Example 5.6 Solution

In [8]:
-((3.21 + 50.0) * 4.184 * (20.3 - 24.9))

1024.1009439999996

$$ \begin{align}
q_{rxn} = - q_{soln} &= - (3.21 g\ \ce{NH4NO3} + 50.0 g\ \ce{H2O}) * 4.184 \frac{J}{g \degree C} * (20.3 \degree C - 24.9 \degree C) \\
&= -(-1.02 kJ) \\[1 ex]
&= 1.02kJ 
\end{align}$$

The positive sign for $q_{reaction}$ means that heat has to be added to the reactants from the environment, making the reaction endothermic.  This assumes that the calorimeter does not provide or absorb heat or allow heat exchange with the outside environment.

<strong style="color:red">Note:</strong> The book gave +1.0 kJ as the answer, but I think that is because they incorporated 20.3 - 24.9 = -4.6 first into the rounding.  I also forgot to mention the assumption that the final solution's specific heat is approximately the same as that of water.  (I also fat fingered the specific heat of water at first, but fixed that when I saw my answer disagreed greatly with the book answer.)

#### Example 5.6 Check Your Learning

In [9]:
-((300 + 3.00) * 4.184 * (-1.05))

1331.1396

#### Example 5.7 Solution

The problem statement is:
> When 3.12 g of glucose, $\ce{C6H12O6}$, is burned in a bomb calorimeter, the temperature of the calorimeter increases from 23.8 °C to 35.6 °C. The calorimeter contains 775 g of water, and the bomb itself has a heat capacity of 893 J/°C. How much heat was produced by the combustion of the glucose sample?

First, let's determine the heat capacity $C_{sys}$ of the system as a whole.
$$ \begin{align}
C_{sys} &= C_{bomb} + C_{water} \\
&= C_{bomb} + m_{water} \times c_{water} \\
&= 893 J/\degree C + 775 g \times 4.184 J/g \degree C 
\end{align}$$
Then multiply by temperature change $\Delta T$ to get the heat energy produced $q_{rel}$.

$$ \begin{align}
q_{rel} &= -q_{rxn} \\
&= C_{sys} \times \Delta T \\
&= (C_{bomb} + m_{water} \times c_{water}) \times (T_f - T_i) \\
&= (893 J/\degree C + 775 g \times 4.184 J/g \degree C)
    \times (35.6 \degree C - 23.8 \degree C)
\end{align} $$

In [10]:
(893 + 775 * 4.184) * (35.6 - 23.8)

48800.08000000001

The solution I get is
$$ q_{rel} = 48.8 \times 10^3 J = 48.4 kJ $$
The book did it in a little different order.
$$ \begin{align}
q_{rxn} &= -(q_{water} + q_{bomb}) \\
&= -[4.184 \times 775 \times (35.6 - 23.8) + 893 \times (35.6 - 23.8)] \\
&= -48.8 kJ
\end{align} $$

In [11]:
-(4.184 * 775 * (35.6 - 23.8) + 893 * (35.6 - 23.8))

-48800.08

#### Example 5.7 Check Your Learning

In [12]:
(784 + 925 * 1 * 4.184) * 8.39

39048.73800000001

$$ q_{rxn} = -39.0 kJ $$

### Chapter 5 &ndash; Exercises